.changeset/silly-students-add.md

```diff
@@ -0,0 +1,7 @@
+---
+"wrangler": patch
+---
+
+polish: Added Wrangler TOML fields
```

Comment:
Doh! I merged this before remembering that we should change this to `feat:`.

Reply:
That's on me I should have changed them both